specs/cloudbeat.spec.yml

```diff
@@ -87,3 +87,9 @@ inputs:
     outputs: *outputs
     command: *command
     isolate_units: true
+  - name: cloudbeat/asset_inventory_gcp
```

Comment:
This file can be moved out of this repository and into cloudbeat's repository, it just needs to be included in the cloudbeat DRA artifact.

Reply:
thanks for pointing this out. i'm going to merge this for now and move the file later.